### PR TITLE
change font size, margin, filter tag's filter name for homepage filters

### LINKED ISSUE #1415: Adjust homepage filter tags
### Overview
We need to adjust the homepage filter tags to make them more consistent with the original Figma design, make the 'looking' tags more understandable, make the font size more accessible, and standardize the class naming according to a discussion about this component's name at a recent dev meeting

### Action Items
- [x] Review the [Figma design for the filter tags (in purple rectangle towards the left)](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=352%3A205)
- [x] Change the " looking: " in the filter tags to " role: "  (i.e. instead of "Looking: PM" it should say "Role: PM." However, please don't change the "Looking" title in the filter dropdown itself
- [x] Make the margin between filter name and number of selected filters ( i.e. Status (2) ) same as in the Figma
- [x] Change the color of 'Clear All' to its color in Figma
- [x] Change the font size of the filter tags to their font size in Figma (make sure the text is still vertically centered on the tag)
- [x] Change the margin between the filter tags to same margin as Figma (or close)
- [ ] Change the CSS class name to '.filter-item' instead of .btn--current-projects-filter-tag (.filter-item name was voted/agreed on at a dev meeting, and this isn't technically a button in the design system). 
   - [ ] Figure out how to make this function similar to a button (clickable with pointer) without using button code. See for reference [Adidas filter tags](https://www.adidas.com/us/women-shoes) with similar functionality, but it's a div

### Resources/Instructions
- [Homepage](https://www.hackforla.org/#)
- [Figma design for the filter tags (in purple rectangle towards the left)](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=352%3A205)

<details>
<summary>Design</summary>
<br>
Screen Shot 2021-04-18 at 4 07 47 AM
</details>

### PR DESCRIPTION
**Pull request by qiqicodes**
Fixed #1415

Changed the following for homepage filters:
1. color of "clear all"
2. filter tags changed from " looking: " to " role: "
3. font size of filter tags
4. margin between filter tags

### CODE REVIEW

**Comment by erikaBell** [comment]
@qiqicodes was there a reason this was put under "draft"? I can't remember what was talked about in the meeting last night on this ticket.

**Comment by qiqicodes** [comment]
@erikaBell after I made the change, the filter toolbar is missing. I am going to debug the branch on Friday night. Thanks for asking! =]

File: _includes/current-projects.html [hunk 1]
```diff
@@ -496,7 +496,7 @@ <h5 class='status-text'>${ project.status }</h5>
   function filterTagComponent(filterName,filterValue){
       return `<button
                   data-filter='${filterName},${filterValue}'
-                  class='btn--current-projects-filter-tag'
+                  class='filter-item'
               >
                   ${filterName}: ${filterValue}
```
Comment (akibrhast) [inline _includes/current-projects.html]: @qiqicodes After speaking with @daniellex0 I realized that changing the filter tag component display from
`looking: development` -> `role:development` is a trivial enough fix. As such please implement that change as was originally requested in the issue

**Review by akibrhast — CHANGES_REQUESTED** [review]

File: _includes/current-projects.html (hunk 1 above)
Comment (qiqicodes) [reply _includes/current-projects.html]: add the requested change to the PR.

**Review by erikaBell — APPROVED** [review]
Looks good! @qiqicodes only request would be to add some 'before' and 'after' photos next time (-: Thanks ~ ~

**Pull request merged**